libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -514,7 +514,13 @@ dictionary OpeningFeeParamsMenu {
     sequence<OpeningFeeParams> values;
 };
 
+enum LspMode {
+    "Breez",
+    "LSPS5",
+};
+
 dictionary LspInformation {
+    LspMode mode;
```

Comment:
Thinking out loud as I don't know yet how this is going to be used. Another alternative is to convert the LspInformation into enum that has either BreezInfo or LSPS5Info as I imagine they would contain different data.
Not an action item for this PR but when the usage is implemented it will become more clear on how to structure this.